Documentation/ApiOverview/UpdateWizards/Creation/Index.rst

```diff
@@ -0,0 +1,74 @@
+.. include:: ../../../Includes.txt
+
+.. _update-wizards-creation-generic:
+
+===============================
+Creating generic update wizards
+===============================
+
+Each update wizard consists of a single PHP file containing a single PHP class. This
+class has to extend :php:`TYPO3\CMS\Install\Updates\AbstractUpdate` and implement the
+abstract methods::
+
+   <?php
+   namespace Vendor\ExtName\Updates;
+
+   use TYPO3\CMS\Install\Updates\AbstractUpdate;
+
+   class ExampleUpdateWizard extends AbstractUpdate
+   {
+       /**
+        * @var string
+        */
+       protected $title = 'Title of this updater';
+
+       /**
+        * Checks whether updates are required.
+        *
+        * @param string &$description The description for the update
+        * @return bool Whether an update is required (TRUE) or not (FALSE)
+        */
+       public function checkForUpdate(&$description)
+       {
+           return true;
+       }
+
+      /**
+       * Performs the accordant updates.
+       *
+       * @param array &$dbQueries Queries done in this update
+       * @param string &$customMessage Custom message
+       * @return bool Whether everything went smoothly or not
+       */
+       public function performUpdate(array &$databaseQueries, &$customMessage)
+       {
+           return true;
+       }
+   }
+
+Property :php:`$title`
+   Can be overwritten to define the title used while rendering the list of available
+   update wizards.
+
+Method :php:`checkForUpdate`
+   Is called to check whether the updater has to run. Therefore a boolean has to be
+   returned. The :php:`$description` provided can be modified as a reference to
+   provide further explanation, in addition to the title.
+
+Method :php:`performUpdate`
+   Is called if the user triggers the wizard. This method should contain, or call,
+   the code that is needed to execute the update. You can use the
+   :php:`$databaseQueries` and :php:`$customMessage` as referenced to provide further
```

Comment:
"[...] as reference to [...]"
* remove "d" from "reference"

Reply:
I've marked these words (I, you, we) and most of https://css-tricks.com/words-avoid-educational-writing/ as "errors" in my editor to prevent writing them.

I'm totally with you. Especially these docs that are more referenced. That would, perhaps, make sense for tuotrials.